runtime/near-vm-logic/tests/test_view_method.rs

```diff
@@ -0,0 +1,32 @@
+use crate::fixtures::get_context;
```

Comment:
Add test for every function that can be prohibited, not just `signed_account_id` and `block_index`. Maybe use macro to avoid code duplication. Later @fckt would need to incorporate it in https://github.com/nearprotocol/nearcore/pull/1347 where he checks for every bindings api very error message that it could return.